scripts/proteinInteractionEBI/README.md

```diff
@@ -0,0 +1,62 @@
+# Scripts for importing ontology dataset from the European Bioinformatics Institute (EMBL-EBI)
+
+This directory stores all scripts used to import datasets from the European Bioinformatics Institute (EMBL-EBI). 
+Here we only import the three subsets of the ontologies: "interaction detection method", "interaction type" and "database citation", which are commonly used in protein-protein interactions. 
+
+## Usage
+
+To generate 'BioOntologySchemaEnum.mcf' which contains all the schemas and "psimi2dcid.txt" which contains paired PSI-MI identifiers and DCID, run:
+
+```bash
+python3 parseEBI.py mi.owl 
+```
+
+If new reference sources which are not properties in dcs occur, 'BioEBINewSource.txt' containing such information will be generated as well.
+
+## Database format
+
+Data is available for download from
+http://psidev.info/groups/controlled-vocabularies. It can also be browsed at https://www.ebi.ac.uk/ols/ontologies/mi. 
+The ontologies dictionary has a tree structure. Note here that one parent node can have multiple child nodes and one child node can have multiple parent nodes as well.
+
+![Tree Structure](./graph/ontologyTree.png)
+
+![Multiple Parent Node](./graph/multipleParent.png)
+
+The original data file (mi.owl) uses "is_a" and "part_of" as the relation property to connect the child node to the parent node, however we don't distinguish these two and a property "specializationOf" is used for the child-parent connection.
```

Comment:
To confirm, are you using part_of? I see you use "relationship".

Reply:
Yes both "part_of" and "is_a" are considered child -> parent linkage, which is the "specializationOf" property in dcs